src/index.html

```diff
@@ -4,12 +4,21 @@
   <meta charset="utf-8">
   <title>Formbox</title>
   <base href="/">
-  <script src="./assets/officejs/office.js"  type="text/javascript"></script>
+  <script src="./assets/officejs/office.js" type="text/javascript"></script>
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
 </head>
-<body>
+
+<body class="container">
```

Comment:
Container in Bootstrap müssen meiner Erfahrung nach als div deklariert werden. Alles andere gibt seltsame Seiteneffekte.
Wenn der Container hier in index.html angelegt wird. Darf man sonst nirgendwo einen Container verwenden.